types/perf_hooks/index.d.ts

```diff
@@ -0,0 +1,66 @@
+export interface Histogram {
+    /**
+     * Returns a `Map` object detailing the accumulated percentile distribution.
+     * @since v11.10.0
+     */
+    readonly percentiles: Map<number, number>;
+    /**
+     * The number of times the event loop delay exceeded the maximum 1 hour event
+     * loop delay threshold.
+     * @since v11.10.0
+     */
+    readonly exceeds: number;
+    /**
+     * The minimum recorded event loop delay.
+     * @since v11.10.0
+     */
+    readonly min: number;
+    /**
+     * The maximum recorded event loop delay.
+     * @since v11.10.0
+     */
+    readonly max: number;
+    /**
+     * The mean of the recorded event loop delays.
+     * @since v11.10.0
+     */
+    readonly mean: number;
+    /**
+     * The standard deviation of the recorded event loop delays.
+     * @since v11.10.0
+     */
+    readonly stddev: number;
+    /**
+     * Resets the collected histogram data.
+     * @since v11.10.0
+     */
+    reset(): void;
+    /**
+     * Returns the value at the given percentile.
+     * @since v11.10.0
+     * @param percentile A percentile value in the range (0, 100].
+     */
+    percentile(percentile: number): number;
+}
+
```

Comment:
This file is `types/perf_hooks/index.d.ts` that introduces a new package `@types/perf_hooks`. Is this what you intend to do?